go.mod

```diff
@@ -36,6 +36,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/gogo/googleapis v1.4.1 // indirect
```

Comment:
@phisco any idea why this dep is being pulled out into its own block?

Reply:
Not sure 🤔 I'll have to check

Reply:
Looks like it is a known issue with `go mod tidy`: https://github.com/golang/go/issues/56471.
Removing this line and running it again results in this separate require block.
However, if manually added to the require block below, `go mod tidy` will only sort it properly. So I'll just move it down and we should keep an eye on these kind of issues. Thanks @hasheddan for pointing it out 🙏